.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,45 @@
+name: Prepare a new release
+
+on:
+  workflow_dispatch:
+    inputs:
+      strategy:
+        description: Valid semver number <x.x.x> or strategy <patch, minor, major>
+        default: "patch"
+        required: false
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Bump the version using input strategy
+        run: yarn version ${{ github.event.inputs.strategy }}
+
+      - name: Update changelog
```

Comment:
TODO: We'll have to test everything on a fork

Reply:
Done